rosidl_typesupport_introspection_cpp/resource/msg__type_support.cpp.em

```diff
@@ -17,11 +18,12 @@
 
 // providing offsetof()
 #include <cstddef>
+#include <vector>
 
-#include <rosidl_generator_c/message_type_support_struct.h>
+#include "rosidl_generator_c/message_type_support_struct.h"
 // this is defined in the rosidl_typesupport_introspection_cpp package and
 // is in the include/rosidl_typesupport_introspection_cpp/impl folder
-#include <rosidl_generator_cpp/message_type_support.hpp>
+#include "rosidl_generator_cpp/message_type_support.hpp"
```

Comment:
While our includes might benefit from more consistency I don't think this needs to happen within this PR.